dts/bindings/sensor/st,iis2dlpc-common.yaml

```diff
@@ -10,3 +10,14 @@ properties:
         This pin defaults to active high when produced by the sensor.
         The property value should ensure the flags properly describe
         the signal that is presented to the driver.
+
+    range:
+      type: int
+      required: false
+      default: 2
```

Comment:
Is 2 g the power-up default?

    default: 2   # sensor power-up default

Reply:
Yes